### DISCUSSION TITLE
Whats mean baseline

### DISCUSSION THREAD

**Discussion by omarwater98** [gmtsar/gmtsar#611] [General]
What does the baseline mean physically, is a distance since the floor or something else?

**Comment by Xiaohua-Eric-Xu** [comment]
Baseline is the distance between the positions of the satellite when its taking the reference and repeat measurement. 

![image](https://user-images.githubusercontent.com/16844111/209897316-aeb5d1bf-353c-4b0d-ac13-11a277cd0eff.png)

**Reply by omarwater98** [reply]
Thanks proffesor
